docs/middleware/ics29-fee/integration.md

```diff
@@ -0,0 +1,83 @@
+<!--
+order: 2
+-->
+
+# Integration
+
+Learn how to configure the Fee Middleware module with IBC applications. The following document only applies for Cosmos SDK chains.  {synopsis}
+
+## Pre-requisite Readings
+
+* [IBC middleware development](../../ibc/middleware/develop.md) {prereq}
+* [IBC middleware integration](../../ibc/middleware/integration.md) {prereq}
+
+The Fee Middleware module, as the name suggests, plays the role of an IBC middleware and as such must be configured by chain developers to route and handle IBC messages correctly.
+For Cosmos SDK chains this setup is done via the `app/app.go` file, where modules are constructed and configured in order to bootstrap the blockchain application.
+
+## Configuring an application stack with Fee Middleware
+
+As mentioned in [IBC middleware development](../../ibc/middleware/develop.md) an application stack may be composed of many or no middlewares that nest a base application. 
+These layers form the complete set of application logic that enable developers to build composable and flexible IBC application stacks.
+For example, an application stack may be just a single base application like `transfer`, however, the same application stack composed with `29-fee` will nest the `transfer` base application
+by wrapping it with the Fee Middleware module.
+
+
+### Transfer
+
+See below for an example of how to create an application stack using `transfer` and `29-fee`.
+The in-line comments describe the execution flow of packets between the application stack and IBC core.
```

Comment:
it's a little bit confusing that you mention app.go two sections above but only show the code snippet here -- maybe either move the sentence from above "For Cosmos SDK chains this setup...." down to the start of this section or move the code snippet up somewhere closer, and indicate that the code snippet is from `app.go`

Reply:
also maybe mention a bit about the ordering of the middleware and how it will affect how the code is called "the bug from evmos"

-- this can also go into the main middleware docs but i dont think it hurts to repeat

Reply:
Meant to leave a reply here earlier... I added an extra sentence to each example section reiterating the `app/app.go` direction. I feel the ordering is covered by the in line comments in the code snippets and should be covered in the middleware docs which are listed as `{prereq}` reading. 

We may want to review the middleware developer/integration docs also to make sure they are up to date!